apps/barista/src/app.ts

```diff
@@ -29,24 +29,40 @@ import { BaPageService } from './shared/page.service';
       '_handleClick($event.target, $event.button, $event.ctrlKey, $event.metaKey)',
   },
 })
-export class BaApp implements OnInit {
+export class BaApp {
   /**
    * @internal
    * The object containing all data needed to display the current page.
    */
-  _currentPage: BaSinglePageContents;
+  _currentPage$ = this.pageService.currentPage;
+
+  /**
+   * @internal
+   * Observable of the current path.
+   */
+  _breadcrumbs$ = this.locationService.currentPath$.pipe(
+    map(path => {
```

Comment:
Can we explicitly type the mapped function parameter?
```suggestion
    map((path: string) => {
```